custom_components/opensprinkler/binary_sensor.py

```diff
@@ -136,9 +136,9 @@ def unique_id(self) -> str:
     def icon(self) -> str:
         """Return icon."""
         if self._program.is_running:
-            return "mdi:timer"
+            return "mdi:timer-outline"
```

Comment:
while testing noticed the deprecation notice in the logs

`2020-07-15 17:49:05 WARNING (MainThread) [frontend.js.latest.202007140] Icon mdi:timer was renamed to mdi:timer-outline, please change your config, it will be removed in version 0.115`

Reply:
actually i am not sure this has been released yet (as i tested it on latest commit in the `core` repo), so I will remove it for now